lab_3/main.py

```diff
@@ -0,0 +1,300 @@
+import argparse
+import logging
+import os
+import csv
+import shutil
+from datetime import datetime, timedelta
+import urllib.request
+import pytz
+
+
+class UserData:
+    """ Class to work with user database """
+    def __init__(self):
+        """
+        Initializes an instance of UserData class.
+        This constructor sets up the argument parser and parses the command line arguments.
+        It also initializes various instance variables used for data processing.
```

Comment:
sorry, I suppose I've confused you a bit. in general __init__  don't need a doc string